templates/devices/device_detail.html

```diff
@@ -486,6 +504,7 @@ <h4>{% trans "10 last sent mails" %}</h4>
                                     <td>{{ mail.subject }}</td>
                                     <td><a href="{% url 'userprofile' mail.sent_by.pk %}">{{ mail.sent_by }}</a></td>
                                     <td><span title="{{ mail.sent_at|date:'c' }}"
+                                              data-timeago
```

Comment:
Do you know why that was previously wrapped inside a `document.ready()` ?

Reply:
It still is.